docs/extend/legacy_plugins.md

```diff
@@ -0,0 +1,91 @@
+<!--[metadata]>
```

Comment:
I think we saw that in another PR as well, because the "old" file is also replaced with a new one. Weird thing is that you see a "move" if you look at the individual commit; https://github.com/docker/docker/pull/25726/commits/a7a70433cabe5eb210ef81ff61f953ab9d9e332d

I just tried what happens if you do two separate commits; one with just the move, and one with adding the new index.md, and it shows the same issue (and also seems to loose the history?) https://github.com/docker/docker/compare/master...thaJeztah:try-move-docs